src/pybind/mgr/balancer/module.py

```diff
@@ -19,6 +19,7 @@
 
 TIME_FORMAT = '%Y-%m-%d_%H:%M:%S'
 
+CRUSH_ITEM_NONE = 0x7fffffff
```

Comment:
can we define it as a class variable of `CRUSHMap` in `mgr_module.py`? as we might want to reference it some where else in future also.